docs/topics/dokka-gradle-configuration-options.md

```diff
@@ -0,0 +1,871 @@
+        </p>
+        <p>Default: <code>true</code></p>
+    </def>
+    <def title="skipDeprecated">
+        <p>Whether to document declarations annotated with <code>@Deprecated</code>.</p>
+        <p>This can be configured on a per-package basis.</p>
+        <p>Default: <code>false</code></p>
+    </def>
+    <def title="suppressGeneratedFiles">
+        <p>Whether to document/analyze generated files.</p>
+        <p>
+            Generated files are expected to be present under the <code>{project}/{buildDir}/generated</code> directory.
+        </p>
+        <p>
+            If set to <code>true</code>, it effectively adds all files from that directory to the
+            <code>suppressedFiles</code> option, so you can configure it manually.
+        </p>
+        <p>Default: <code>true</code></p>
+    </def>
+    <def title="jdkVersion">
+        <p>The JDK version to use when generating external documentation links for Java types.</p>
+        <p>
+            For example, if you use <code>java.util.UUID</code> in some public declaration signature,
+            and this option is set to <code>8</code>, Dokka generates an external documentation link
+            to <a href="https://docs.oracle.com/javase/8/docs/api/java/util/UUID.html">JDK 8 Javadocs</a> for it.
+        </p>
+        <p>Default: JDK 8</p>
+    </def>
+    <def title="languageVersion">
+        <p>
+            <a href="https://kotlinlang.org/docs/compatibility-modes.html">The Kotlin language version</a>
+            used for setting up analysis and <a href="https://kotlinlang.org/docs/kotlin-doc.html#sample-identifier">@sample</a>
+            environment.
+        </p>
+        <p>By default, the latest language version available to Dokka's embedded compiler is used.</p>
+    </def>
+    <def title="apiVersion">
+        <p>
+            <a href="https://kotlinlang.org/docs/compatibility-modes.html">The Kotlin API version</a>
+            used for setting up analysis and <a href="https://kotlinlang.org/docs/kotlin-doc.html#sample-identifier">@sample</a>
+            environment.
+        </p>
+        <p>By default, it is deduced from <code>languageVersion</code>.</p>
+    </def>
+    <def title="sourceRoots">
+        <p>
+            The source code roots to be analyzed and documented.
+            Acceptable inputs are directories and individual <code>.kt</code> / <code>.java</code> files.
+        </p>
+        <p>By default, source roots are deduced from information provided by the Kotlin Gradle plugin.</p>
+    </def>
+    <def title="classpath">
+        <p>The classpath for analysis and interactive samples.</p>
+        <p>This is useful if some types that come from dependencies are not resolved/picked up automatically.</p>
+        <p>This option accepts both <code>.jar</code> and <code>.klib</code> files.</p>
+        <p>By default, the classpath is deduced from information provided by the Kotlin Gradle plugin.</p>
```

Comment:
This is not critical, but sometimes the default is introduced with "Default:" and sometimes with "By default", which could be more uniform.